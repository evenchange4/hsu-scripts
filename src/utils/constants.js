// @flow
import path from 'path';

export const babelrcPath = path.resolve(__dirname, '../../.babelrc');

// Note: Build
export const DEFAULT_ES_DIR = 'es';
export const DEFAULT_CJS_DIR = 'lib';
export const DEFAULT_IGNORE = ['__tests__', '**/*.test.js', '**/*.example.js'];

// Note: flow
export const DEFAULT_CONCURRENT_FILES = 5;
export const DEFAULT_INCLUDE_GLOB = ['**/*.js'];
export const DEFAULT_EXCLUDE_GLOB = [
  'node_modules/**',
  'public/**',
  '.next/**',
  'coverage/**',
  'storybook-static/**',
  'flow-typed/**',
  'lib/**',
  'es/**',
];
export const DEFAULT_TYPE = ['text'];
export const DEFAULT_THRESHOLD = 90;
