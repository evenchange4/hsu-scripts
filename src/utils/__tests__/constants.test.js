// @flow
import * as constants from '../constants';

it('should return constants correctly', () => {
  expect(constants).toMatchSnapshot();
});
