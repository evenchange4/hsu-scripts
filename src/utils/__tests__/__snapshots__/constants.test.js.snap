// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should return constants correctly 1`] = `
Object {
  "DEFAULT_CJS_DIR": "lib",
  "DEFAULT_CONCURRENT_FILES": 5,
  "DEFAULT_ES_DIR": "es",
  "DEFAULT_EXCLUDE_GLOB": Array [
    "node_modules/**",
    "public/**",
    ".next/**",
    "coverage/**",
    "storybook-static/**",
    "flow-typed/**",
    "lib/**",
    "es/**",
  ],
  "DEFAULT_IGNORE": Array [
    "__tests__",
    "**/*.test.js",
    "**/*.example.js",
  ],
  "DEFAULT_INCLUDE_GLOB": Array [
    "**/*.js",
  ],
  "DEFAULT_THRESHOLD": 90,
  "DEFAULT_TYPE": Array [
    "text",
  ],
  "__esModule": true,
  "babelrcPath": "/__dirname/../../.babelrc",
}
`;
